app/auth.py

```diff
@@ -0,0 +1,63 @@
+from flask import current_app as app, flash, redirect, url_for, render_template
+from flask_login import login_user, current_user, logout_user, login_required
+from werkzeug.security import generate_password_hash, check_password_hash
+from app import db
+from app.forms import RegisterForm, LoginForm
+from app.models import User
+
+
+@app.route('/register', methods=["GET", "POST"])
+def register():
+    form = RegisterForm()
+    if form.validate_on_submit():
+        if User.query.filter_by(email=form.email.data).first():
+            # User already exists
+            flash("You've already signed up with that email, log in instead!")
```

Comment:
Nit: Maybe think about "kinder" wording and eliminating the `!`, e.g.:
```suggestion
            flash("An account already exists with that email. Please log in instead.")
```

Reply:
Haha. I got too excited.